### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
Corrected the grammar of the document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ shell with:
 > python x.py build
 ```
 
-Currently building Rust only works with some known versions of Visual Studio. If
+Currently, building Rust only works with some known versions of Visual Studio. If
 you have a more recent version installed the build system doesn't understand
-then you may need to force rustbuild to use an older version. This can be done
+then you may need to force rust build to use an older version. This can be done
```
Comment (jonas-schievink) [inline README.md]: rustbuild is correct, it's the name of the build tool that bootstraps the compiler and builds everything with Cargo

File: README.md [hunk 2]
```diff
@@ -159,7 +159,7 @@ $ ./configure
 $ make && sudo make install
 ```
 
-When using the configure script, the generated `config.mk` file may override the
+When using the configure script, the generated `config.mk` the file may override the
```
Comment (jonas-schievink) [inline README.md]: That's an extra `the` - it was correct before the change

File: README.md [hunk 3]
```diff
@@ -180,9 +180,9 @@ the ABI used. I.e., if the ABI was `x86_64-pc-windows-msvc`, the directory will
 [notes]: #notes
 
 Since the Rust compiler is written in Rust, it must be built by a
-precompiled "snapshot" version of itself (made in an earlier state of
+precompiled "snapshot" version of itself (made in an earlier stage of
 development). As such, source builds require a connection to the Internet, to
-fetch snapshots, and an OS that can execute the available snapshot binaries.
+fetch snapshots and an OS that can execute the available snapshot binaries.
```
Comment (jonas-schievink) [inline README.md]: This was an Oxford comma, which is also correct and makes this more readable, imo

Comment (dhiru30) [reply README.md]: @jonas-schievink  Thanks, I have reverted it.

File: README.md (hunk 2 above)
Comment (dhiru30) [reply README.md]: @jonas-schievink  Yes Correct. I have reverted it.

File: README.md (hunk 1 above)
Comment (dhiru30) [reply README.md]: @jonas-schievink  Thanks for correction. I have reverted it.

**Pull request merged**